src/pages/groupComparison/ProteinEnrichments.tsx

```diff
@@ -21,31 +23,39 @@ export default class ProteinEnrichments extends React.Component<IProteinEnrichme
         this.props.store.setProteinEnrichmentProfile(m);
     }
 
+    private readonly enrichmentAnalysisGroups = remoteData({
+        await: () => [this.props.store._activeGroupsOverlapRemoved],
+        invoke: () => {
+            const groups = _.map(this.props.store._activeGroupsOverlapRemoved.result, group => {
```

Comment:
you can use the native map method on array.  if this is an object, then yeah, lodash map is helpful